index.html

```diff
@@ -83,31 +88,31 @@ <h2>ПОПУЛЯРНЫЕ ТОВАРЫ:</h2>
       <ul>
         <li><a href="#">КУПИТЬ</a>
           <a href="#">В ЗАКЛАДКУ</a>
-          <img src="" alt="">
+          <img src="img/perfor1.jpg" width="184" height="164" alt="Перфоратор-1">
```

Comment:
Альт дожжен быть осмыслен. В этом случае лучше указать название перфоратора
```suggestion
          <img src="img/perfor1.jpg" width="184" height="164" alt="Перфоратор BOSCH BFG 9000">
```

Касается всех картинок перфораторов